webpack.config.js

```diff
@@ -7,10 +8,15 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const CompressionPlugin = require("compression-webpack-plugin");
 const ESLintPlugin = require('eslint-webpack-plugin');
 const CockpitPoPlugin = require("./src/lib/cockpit-po-plugin");
+const CockpitRsyncPlugin = require("./src/lib/cockpit-rsync-plugin");
 
 /* A standard nodejs and webpack pattern */
 const production = process.env.NODE_ENV === 'production';
 
+// Obtain package name from package.json
+const data = fs.readFileSync('./package.json', { encoding:'utf8', flag:'r' });
```

Comment:
Let's add spaces after `:`.

But I don't think you'll need these options at all -- e.g. [here](https://github.com/cockpit-project/cockpit/blob/main/.github/workflows/npm-install.yml#L90) we read package.json without any options.

Please either eliminate the `data` const name (preferred), or rename it to `package_json_str` or so.